support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -366,6 +366,13 @@ export function ThreeTierLanding(): JSX.Element {
 				contributionTypeKeyOverride ?? tierPlanPeriod
 			].charges[countryGroupId];
 
+		if (cardTier === 1) {
+			tierPlanCountryCharges = {
+				...tierPlanCountryCharges,
+				price: recurringAmount,
+			};
+		}
```

Comment:
I think we should refactor how we transform the cards, and just read them from the catalog. But this will at least give people some functionality to test out.